build.gradle

```diff
@@ -8,9 +8,9 @@ plugins {
 
 project.description 'Gradle project versioning plugin based on Git tags.'
 project.group = 'com.productmadness'
-project.version = '0.1.0'
+project.version = '1.0.0'
```

Comment:
What does this project.verision do?
Given that it is based on Github Tag, why do we have this version?

Reply:
Ha wait, Gradle Versioning Plugin probably does not use Gradle Versioning Plugin to set its own version!

Reply:
yeah, lol :)